shared/templates/sshd_lineinfile/bash.template

```diff
@@ -11,8 +11,9 @@ mkdir -p /etc/ssh/sshd_config.d
 touch /etc/ssh/sshd_config.d/hardening
 {{{ lineinfile_absent("/etc/ssh/sshd_config", line_regex, insensitive=true) }}}
 {{{ lineinfile_absent_in_directory("/etc/ssh/sshd_config.d", line_regex, insensitive=true) }}}
+{{%- set hardening_config_basename = "60-complianceascode-hardening.conf" %}}
```

Comment:
I think it's ok to have a higher number here since our remediation would remove any offending occurrences from any other file including the ones with lower number that would take precedence in the configuration loading. If *not* then I'd suggest we use a lower number.

Reply:
I somehow thought that it is the last directive that determines what's going to happen, but it's the exact opposite - as documented, and also as verified.
I will therefore change the value of `60-comp...` to a universal `00-comp...`.